app/views/admin/articles/_article.html.haml

```diff
@@ -2,7 +2,12 @@
   %td.govuk-table__cell
     = article.description
   %td.govuk-table__cell
-    = article.published_at&.strftime("%d-%m-%Y")
+    - if article.published?
+      = article.published_at&.strftime("%d-%m-%Y")
+    - elsif article.unpublished?
+      = article.unpublished_at&.strftime("%d-%m-%Y")
+    - else
+      = article.created_at&.strftime("%d-%m-%Y")
```

Comment:
If you add an initializer: `config/initializers/time_formats.rb` with this content:
```ruby
Time::DATE_FORMATS[:date_only] = "%d-%m-%Y"  # 04-11-2021
```
Here you could do:
```ruby
    - if article.published?
      = article.published_at&.to_s(:date_only)
    - elsif article.unpublished?
      = article.unpublished_at&.to_s(:date_only)
    - else
      = article.created_at&.to_s(:date_only)
```
Which will aid consistency and better communicate intent.